UploadFile/Page/Pag50410.BCPUpload.al

```diff
@@ -0,0 +1,32 @@
+page 50410 "BCP Upload"
+{
+    PageType = ListPart;
+    ApplicationArea = All;
+    UsageCategory = None;
+    SourceTable = "BCP Upload";
+
+    layout
+    {
+        area(Content)
+        {
+            repeater("File Upload")
+            {
+
+                field("BCP File Name"; Rec."BCP File Name")
+                {
+                    ApplicationArea = All;
+                    ToolTip = 'Filename';
+                    trigger OnDrillDown()
+                    var
+                        downloadFunc: Codeunit "BCP UploadFile";
```

Comment:
In general variable name should be same as object named. In this case UploadFile. But, I am sure this is not best name for codeunit in general.

Reply:
You're right, i'll make changes to that.